x-pack/plugin/eql/qa/common/src/main/resources/test_sample.toml

```diff
@@ -3,11 +3,14 @@ name = "anyRulesWithOneDifferentNameKey"
 query = '''
```

Comment:
At the top of this file, at least, please add a comment describing this syntax (array of arrays in case of multiple possible solutions).